aws/resource_aws_cloudwatch_event_target.go

```diff
@@ -216,6 +216,10 @@ func resourceAwsCloudWatchEventTarget() *schema.Resource {
 							Type:     schema.TypeMap,
 							Optional: true,
 							Elem:     &schema.Schema{Type: schema.TypeString},
+							ValidateFunc: validation.All(
+								MapLenBetween(0, 10),
+								MapKeyNotMatch(regexp.MustCompile(`^AWS.*$`), "input_path must not start with \"AWS\""),
```

Comment:
Slight nit, the error message should say it can't start with `"AWS."`
```suggestion
								MapKeyNotMatch(regexp.MustCompile(`^AWS.*$`), "input_path must not start with \"AWS.\""),
```

Reply:
@gdavison cheers for the feedback, will start working these action points. On this one though,  from the docs it looks like the restrictions is just AWS (without the dot)? Is that not correct?
```
	// Map of JSON paths to be extracted from the event. You can then insert these
	// in the template in InputTemplate to produce the output to be sent to the
	// target.
	//
	// InputPathsMap is an array key-value pairs, where each value is a valid JSON
	// path. You can have as many as 10 key-value pairs. You must use JSON dot notation,
	// not bracket notation.
	//
	// The keys can't start with "AWS".
```

Reply:
Ah, I misread that.